standalone/README.md

```diff
@@ -0,0 +1,27 @@
+To convert aa LCIO file to EDM4hep either a LCIO file where all collections are present in every event
+or a file containing a list of all collections in the file are needed. 
+This list can be obtained by executing 'check_missing_cols' from the LCIO package with the '--minimal' flag.
+
```

Comment:
Can you put a small sentence and an example for the format of these files here as well? So that people know how it should look like if they want to create one from scratch.

Also add another sentence, that this file is simply used to patch missing collections and cannot be used to select which collections will be converted.